RT/dt_evolve_hpsi.f90

```diff
@@ -61,7 +62,7 @@ subroutine dt_evolve_hpsi
     call update(zfac,ztpsi(:,:,tid),zu(:,ib,ik))
 
 #ifdef ARTED_CURRENT_OPTIMIZED
-    call current_omp_KB_ST(ib,ik,zu(:,ib,ik))
+    if(flag_current) call current_omp_KB_ST(ib,ik,zu(:,ib,ik))
```

Comment:
Yes, you are right. I changed the propagator to etrs. In the single-cell calculation, I use "dt_evolve_etrs_omp_KB" instead of "dt_evolve_omp_KB". 

The new propagation scheme calls "dt_evolve_hpsi" two or three times in a single time-step. Thus, the computation of a partial current in "dt_evolve_hpsi" is not required in every call but the last call of "dt_evolve_hpsi".